content/docs/lists-and-keys.md

```diff
@@ -42,15 +42,15 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/GjPyQr?editors=0011)
+[**कोडपेन पर इसे आजमाएं**](https://codepen.io/gaearon/pen/GjPyQr?editors=0011)
```

Comment:
`कोडपेन` => `CodePen`
I think we should not translate this, as its name of a website. Any thoughts?